src/mumble/MainWindow.cpp

```diff
@@ -3040,11 +3048,15 @@ void MainWindow::trayAboutToShow() {
 
 	qmTray->clear();
 	if (top) {
+		qmTray->addAction(qaShow);
+		qmTray->addSeparator();
```

Comment:
Since the separator is used to group related actions together (e.g. audio mute and audio deaf), I think that we should leave `qaShow` together with `qaQuit`.

What do you think?